great-idea/services.html

```diff
@@ -4,103 +4,121 @@
 <head>
   <meta charset="utf-8">
 
-  <title>Great Idea!</title>
+  <title>Services</title>
```

Comment:
Title should include....Great Idea! - Services.
Not an issue today but this is where you would want to pay attention to those little details when given instruction by a customer or boss.  The detail can make a difference between a happy boss/customer or a dissatisfied one.